spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/SarifBugReporter.java

```diff
@@ -17,29 +19,42 @@ public SarifBugReporter(Project project) {
     @Override
     public void finish() {
         try {
-            JSONWriter jsonWriter = new JSONWriter(outputStream);
-            jsonWriter.object();
-            jsonWriter.key("version").value("2.1.0").key("$schema").value(
+            JsonWriter jsonWriter = new JsonWriter(outputStream);
+            jsonWriter.beginObject();
+            jsonWriter.name("version").value("2.1.0").name("$schema").value(
                     "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json");
             processRuns(jsonWriter);
             jsonWriter.endObject();
             getBugCollection().bugsPopulated();
+        } catch (IOException e) {
+            e.printStackTrace(System.err);
```

Comment:
I assume it's better to throw [UncheckedIOException](https://docs.oracle.com/javase/8/docs/api/java/io/UncheckedIOException.html) instead?
At least, we cannot ignore this exception like this, because we surely have problem in generated SARIF report file.

Reply:
Here we do not need to invoke `e.printStackTrace(System.err);`. Who catches the error will print it.

And when we need to log, we use SLF4J as a logging facade so please do not use `System.err` directly.

Reply:
Removed printStackTrace to let caller to log the error.